bin/nyc.js

```diff
@@ -50,7 +51,7 @@ if (argv._[0] === 'report') {
     NYC_CONFIG: JSON.stringify(config),
     NYC_CWD: process.cwd(),
     NYC_ROOT_ID: nyc.rootId,
-    BABEL_DISABLE_CACHE: 1,
+    BABEL_DISABLE_CACHE: config['babel-cache'],
```

Comment:
This is my only doubt about this change: this variable was a `number` before, and now it will be a `boolean`. 

Should the config accept a number? or default to a string number (`'1'`)?